### PR TITLE
updated the AMREX_CUDA_ARCH vvariable documentation

### PR DESCRIPTION
**Pull request by srichers**
## Summary

Updated the documentation for the AMREX_CUDA_ARCH variable to match the capitalization and possible arguments that currently work.

## Checklist

The proposed changes:
- [ ] fix a bug or incorrect behavior in AMReX
- [ ] add new capabilities to AMReX
- [ ] changes answers in the test suite to more than roundoff level
- [ ] are likely to significantly affect the results of downstream AMReX users
- [X] include documentation in the code and/or rst files, if appropriate

### CODE REVIEW

**Comment by WeiqunZhang** [comment]
I don't think the proposed change is correct.  The CMake systems takes `AMREX_CUDA_ARCH` with all capitals from the environment, but the CMake variable passed in the command line is called `AMReX_CUDA_ARCH` with a lower case `e`.

@ax3l @sayerhs Is that right?

File: Docs/sphinx_documentation/source/GPU.rst [hunk 1]
```diff
@@ -207,7 +207,7 @@ check the :ref:`table <tab:cmakecudavar>` below.
    +------------------------------+-------------------------------------------------+-------------+-----------------+
    | Variable Name                | Description                                     | Default     | Possible values |
    +==============================+=================================================+=============+=================+
-   | AMReX_CUDA_ARCH              |  CUDA target architecture                       | Auto        | User-defined    |
+   | AMREX_CUDA_ARCH              |  CUDA target architecture                       | Auto        | User-defined    |
```
Comment (ax3l) [inline Docs/sphinx_documentation/source/GPU.rst:210]: This is not an environment variable, but a CMake option (variable) for AMReX.
CMake options are case sensitive and use the original spelling of AMReX.

Please see line 260 for the environment variable. The environment variable is all-caps (for platform compatibility) and changes the default of the `AMReX_CUDA_ARCH` CMake variable, if found.

This workflow is standard for CLI tools in Unix & Windows, thus we implemented this in AMReX as well.

File: Docs/sphinx_documentation/source/GPU.rst [hunk 2]
```diff
@@ -244,24 +244,23 @@ check the :ref:`table <tab:cmakecudavar>` below.
 
 
 The target architecture to build for can be specified via the configuration option
-``-DAMReX_CUDA_ARCH=<target-architecture>``, where ``<target-architecture>`` can be either
-the name of the NVIDIA GPU generation, i.e. ``Turing``, ``Volta``, ``Ampere``, ``...`` , or its
-`compute capability <https://developer.nvidia.com/cuda-gpus>`_, i.e. ``10.0``, ``9.0``,  ``...`` .
+``-DAMREX_CUDA_ARCH=<target-architecture>``, where ``<target-architecture>`` is the
```
Comment (ax3l) [inline Docs/sphinx_documentation/source/GPU.rst:247]: Same here, please do not change this.

File: Docs/sphinx_documentation/source/GPU.rst [hunk 3]
```diff
@@ -244,24 +244,23 @@ check the :ref:`table <tab:cmakecudavar>` below.
...
+`compute capability <https://developer.nvidia.com/cuda-gpus>`_ of the NVIDIA GPU genertaion, i.e. ``60`` for 6.0, ``70`` for 7.0,  ``...`` .
 For example, on Cori GPUs you can specify the architecture as follows:
 
 .. highlight:: console
 
 ::
 
-   cmake [options] -DAMReX_GPU_BACKEND=CUDA -DAMReX_CUDA_ARCH=Volta /path/to/amrex/source
+   cmake [options] -DAMReX_GPU_BACKEND=CUDA -DAMREX_CUDA_ARCH=70 /path/to/amrex/source
```
Comment (ax3l) [inline Docs/sphinx_documentation/source/GPU.rst:255]: Please see above.

File: Docs/sphinx_documentation/source/GPU.rst [hunk 4]
```diff
@@ -244,24 +244,23 @@ check the :ref:`table <tab:cmakecudavar>` below.
...
 
 
 If no architecture is specified, CMake will default to the architecture defined in the
 *environment variable* ``AMREX_CUDA_ARCH`` (note: all caps).
 If the latter is not defined, CMake will try to determine which GPU architecture is supported by the system.
 If more than one is found, CMake will build for all of them.
 If autodetection fails, a list of "common" architectures is assumed.
-`Multiple CUDA architectures <https://cmake.org/cmake/help/latest/module/FindCUDA.html#commands>`__ can also be set manually as semicolon-separated list, e.g. ``-DAMReX_CUDA_ARCH=7.0;8.0``.
+`Multiple CUDA architectures <https://cmake.org/cmake/help/latest/module/FindCUDA.html#commands>`__ can also be set manually as semicolon-separated list, e.g. ``-DAMReX_CUDA_ARCH=70;80``.
```
Comment (ax3l) [inline Docs/sphinx_documentation/source/GPU.rst:263]: Both works, but if you follow the link quoted here, the documented workflow is to use the dotted version (it's also less cryptic). Please do not change here.

**Comment by ax3l** [comment]
Hi @srichers,

Thank you so much for the PR. I added a couple of inline comments to clarify. Please don't hesitate to follow up, if you have any questions, hope this helps :)

**Comment by srichers** [comment]
Thank you for your comments and clarification!

**Comment by ax3l** [comment]
You are welcome, thank you for contributing! :)

**Pull request closed without merging**